### PR TITLE
[ru] Localize `Immutable infrastructure`

### LINKED ISSUE #3119: [ru] Localize `Immutable Infrastructure`
### Description

This Issue is to localize [`Immutable Infrastructure`](https://github.com/cncf/glossary/blob/main/content/en/immutable-infrastructure.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by kirkonru**
### Describe your changes
Localize `Immutable infrastructure`

### Related issue number or link (ex: `resolves #issue-number`)
resolves #3119 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR uses the **dev-ru** branch as a source/target.
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/immutable-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Неизменяемость может быть реализована с помощью автоматического процесса, который перезаписывает несанкционированные изменения, или 
+с помощью системы, которая вообще не позволяет вносить изменения. 
+Контейнеры — хороший пример неизменяемой инфраструктуры. 
+Внести в них постоянные изменения можно только  
+создав новую версию контейнера или воссоздав существующий контейнер из его образа.
+
+Предотвращая или выявляя несанкционированные изменения, 
+неизменяемые инфраструктуры облегчают обнаружение угроз. 
+Управлять такой системой гораздо проще, 
+поскольку администраторы могут делать определенные предположения о ней.
```
Comment (shurup) [inline content/ru/immutable-infrastructure.md:20]: A bit less precise, but it gives a better understanding (IMHO).
```suggestion
поскольку администраторам легче предугадывать её поведение. 
```

File: content/ru/immutable-infrastructure.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+с помощью системы, которая вообще не позволяет вносить изменения. 
+Контейнеры — хороший пример неизменяемой инфраструктуры. 
+Внести в них постоянные изменения можно только  
+создав новую версию контейнера или воссоздав существующий контейнер из его образа.
+
+Предотвращая или выявляя несанкционированные изменения, 
+неизменяемые инфраструктуры облегчают обнаружение угроз. 
+Управлять такой системой гораздо проще, 
+поскольку администраторы могут делать определенные предположения о ней. 
+Ведь они знают, что никто не мог внести в нее изменения (в т.ч. ошибочные), о которых к тому же забыл сообщить.
```
Comment (shurup) [inline content/ru/immutable-infrastructure.md:21]:
```suggestion
Ведь они знают, что никто не мог внести в нее изменения (в т.ч. ошибочные), о которых, к тому же, забыл сообщить. 
```

File: content/ru/immutable-infrastructure.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Контейнеры — хороший пример неизменяемой инфраструктуры. 
+Внести в них постоянные изменения можно только  
+создав новую версию контейнера или воссоздав существующий контейнер из его образа.
+
+Предотвращая или выявляя несанкционированные изменения, 
+неизменяемые инфраструктуры облегчают обнаружение угроз. 
+Управлять такой системой гораздо проще, 
+поскольку администраторы могут делать определенные предположения о ней. 
+Ведь они знают, что никто не мог внести в нее изменения (в т.ч. ошибочные), о которых к тому же забыл сообщить. 
+Неизменяемая инфраструктура неразрывно связана с [инфраструктурой-как-код](/ru/infrastructure-as-code/)
```
Comment (shurup) [inline content/ru/immutable-infrastructure.md:22]:
```suggestion
Неизменяемая инфраструктура неразрывно связана с [инфраструктурой-как-код](/ru/infrastructure-as-code/), 
```

**Review by shurup — COMMENTED** [review]
Small suggestions from me.

**Review by tym83 — APPROVED** [review]
LGTM

**Review by shurup — APPROVED** [review]

**Pull request merged**